website/sidebars.json

```diff
@@ -1,10 +1,9 @@
 {
   "docs": {
-    "Docusaurus": ["doc1"],
-    "First Category": ["doc2"],
-    "Second Category": ["doc3"]
-  },
-  "docs-other": {
-    "First Category": ["doc4", "doc5"]
+    "Overview": ["introduction"],
+    "ADRs": [
```

Comment:
wow :rocket:

Reply:
yeah, turns out if the pages don't exist the sidebar doesn't render ... always wondered what this file was for ha